workflow-templates/shellcheck.yml

```diff
@@ -0,0 +1,29 @@
+# Run ShellCheck on PRs and $default-branch
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - $default-branch
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    strategy:
+      matrix:
+        os: ["ubuntu", "macos"]
+    uses: openedx/.github/.github/workflows/shellcheck.yml@master
+    with:
+      # For details on the meaning of each of these arguments, see:
+      # https://github.com/openedx/.github/blob/master/.github/workflows/shellcheck.yml
+      root-dirs: "./scripts ./.github"
```

Comment:
This seems like a strange default for the template, I'd love for it to default to the whole repo, in the reusable workflow and then this could just be a commented value.

Reply:
Agreed. I flipped from this being an include-list to an exclude-list.